oc/oc-netobserv-flows

```diff
@@ -9,9 +9,25 @@ then
   filter=""
 fi
 
+copyOutput="true"
 trap cleanup EXIT
 
 setup flows $filter
 
 echo "\nRunning network-observability-cli get-flows... "
-${BASH_SOURCE%/*}/network-observability-cli --nodes $nodes --ports $ports ${filter:+"--filter" "$filter"} get-flows
+oc run \
+  -n netobserv-cli \
+  collector \
+  --image=quay.io/jpinsonn/network-observability-cli:dev \
```

Comment:
This image will be updated once gRPC is added to FLP and included in eBPF agent image